### PR TITLE
Reduce logging level

### PR DESCRIPTION
**Pull request by torkleyy**

### CODE REVIEW

File: factory/src/factory.rs [hunk 1]
```diff
@@ -1120,7 +1120,7 @@ where
     let heaps = heaps.into_iter().collect::<SmallVec<[_; 16]>>();
     let types = types.into_iter().collect::<SmallVec<[_; 32]>>();
 
-    log::info!("Heaps: {:#?}\nTypes: {:#?}", heaps, types);
+    log::trace!("Heaps: {:#?}\nTypes: {:#?}", heaps, types);
```
Comment (Frizi) [inline factory/src/factory.rs]: If queues and above are debug, i think this should be as well. It's one time so not a big deal

File: rendy/examples/quads/main.rs [hunk 2]
```diff
@@ -512,14 +512,14 @@ where
...
-            log::info!("Acquire {:?} : {:#?}", stages, barriers);
+            log::trace!("Acquire {:?} : {:#?}", stages, barriers);
             encoder.pipeline_barrier(stages, hal::memory::Dependencies::empty(), barriers);
         }
         encoder.dispatch(QUADS, 1, 1);
 
         {
             let (stages, barriers) = gfx_release_barriers(ctx, &*buffers, None);
-            log::info!("Release {:?} : {:#?}", stages, barriers);
+            log::trace!("Release {:?} : {:#?}", stages, barriers);
```
Comment (Frizi) [inline rendy/examples/quads/main.rs]: i think we actually want `info`s here, this is an example code

**Comment by torkleyy** [comment]
Resolved, would be great if we could get this into a new patch version.

**Comment by torkleyy** [comment]
bors r=Frizi

**Comment by torkleyy** [comment]
Seriously Travis, why?
bors retry

**Pull request merged**